packages/client/src/utils/getTestClient.ts

```diff
@@ -60,7 +60,7 @@ export async function getTestClient(schemaDir?: string, printWarnings?: boolean)
     relativeEnvPaths,
     datasourceNames: config.datasources.map((d) => d.name),
     activeProvider,
-    dataProxy: false,
+    dataProxy: !!process.env.DATA_PROXY,
```

Comment:
```suggestion
    dataProxy: Boolean(process.env.DATA_PROXY),
```
If it's still needed, using `Boolean()` is an easier one to read